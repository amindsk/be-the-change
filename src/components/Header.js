//To create an app component
import React from 'react';

//Header component
const Header = () => (
    //JSX of the component
    <header>
        <h1>Skill Editor</h1>
    </header>
);

//Exporting component
export default Header;