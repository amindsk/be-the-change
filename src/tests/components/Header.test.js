//Importing react for component rendering
import React from 'react';
//Importing shallow renderer to create snapshot of component
import { shallow } from 'enzyme';
//Importing Header component
import Header from '../../components/Header';

//Matches with snapshot and findout if component render correctly
test('Should render Header correctly', () => {
    //Creates snapshot of hte component
    const wrapper = shallow(<Header />);
    //If snapshots matches
    expect(wrapper).toMatchSnapshot();
});