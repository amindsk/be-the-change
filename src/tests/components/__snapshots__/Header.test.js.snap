// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render Header correctly 1`] = `
<header>
  <h1>
    Skill Editor
  </h1>
</header>
`;
